Library/UIViewController-Preparation.swift

```diff
@@ -43,16 +44,21 @@ extension UIViewController {
     self.bindViewModel()
   }
 
+  internal func ksr_viewWillAppear(_ animated: Bool) {
+    self.ksr_viewWillAppear(animated)
+    self.bindStyles()
+  }
```

Comment:
heyyoo, putting `bindStyles` here may cause a host of unexpected changes. We'd have to do an audit to make sure that only non-changing properties are in here, otherwise set in `bindViewModel` (which we should do anyway!). The issue with the project page close button turning white after closing the pledge is because it is being set to white in `bindStyles`. Are we sure about this change? If so, definitely requires a full audit of all `bindStyles`.